propagation/http_trace_context_propagator.go

```diff
@@ -31,7 +31,7 @@ import (
 const (
 	supportedVersion  = 0
 	maxVersion        = 254
-	traceparentHeader = "traceparent"
+	TraceparentHeader = "Traceparent"
```

Comment:
nit:s/Traceparent/traceparent/ - As per the spec header name is all lowercase.

Reply:
Hmm. This is not how it comes through on the other side though: https://github.com/open-telemetry/opentelemetry-go/pull/156/files#diff-55d5b1b76d174a46c4ee33c7f594aa60R37.

Go runs headers through CanonicalHeaderKey as per: https://golang.org/pkg/net/http/#Header

Reply:
Hmm, that is interesting. This means interop test between Go and other language would fail.
If the Header is modified/accessed directly (without Add/Get) then you could use lowercase but that be against the norm. Alternatively, modify W3C spec to accept "Traceparent" or "traceparent". 
/cc @SergeyKanzhelev

Reply:
@freeformz , Let merge this for now. We'll open an issue for addressing this in another PR.